hil_extension/hil.yml

```diff
@@ -0,0 +1 @@
+url: http://127.0.0.1:5000
```

Comment:
It's to keep information about the HIL server. We connect to this address. And we need to know this address upfront.